packages/grant-explorer/src/features/common/styles.ts

```diff
@@ -80,17 +80,18 @@ export const Badge = tw.div<{
   color?: keyof typeof colorMap;
   rounded?: keyof typeof roundedMap;
   disabled?: boolean;
+  flex?: boolean;
 }>`
   font-mono
   text-xs
   text-gray-900
   bg-gray-100
   whitespace-nowrap
-  inline-flex
   items-center
   justify-center
   px-2
   py-1.5
+  ${(p) => (p.flex ?? true ? "inline-flex" : "")}
```

Comment:
Seems unnecessary with a prop for this. I would rather just use the className inline-flex.

Reply:
the inline flex is used by default which breaks truncation, this prop adds a way to remove it, I don't want to risk making changes here and potentially break callers everywhere, either way I have deleted it and used a simpler approach